Source/HTTP/WinHttp/winhttp_http_task.cpp

```diff
@@ -13,27 +13,102 @@
 
 using namespace xbox::httpclient;
 
+
+HC_PERFORM_ENV::HC_PERFORM_ENV()
+{
+    m_proxyType = get_ie_proxy_info(proxy_protocol::https, m_proxyUri);
```

Comment:
Maybe make a note of future work to split out https proxy vs proxy for other protocols